mmcv/ops/merge_cells.py

```diff
@@ -95,8 +95,19 @@ def _resize(self, x, size):
         elif x.shape[-2:] < size:
             return F.interpolate(x, size=size, mode=self.upsample_mode)
         else:
-            assert x.shape[-2] % size[-2] == 0 and x.shape[-1] % size[-1] == 0
-            kernel_size = x.shape[-1] // size[-1]
+            if x.shape[-2] % size[-2] != 0 or x.shape[-1] % size[-1] != 0:
+                h, w = x.shape[-2:]
+                h_t, w_t = size
+                padding_h = (h // h_t + 1) * h_t - h
+                padding_w = (w // w_t + 1) * w_t - w
+                padding_left = padding_w // 2
+                padding_right = padding_w - padding_left
+                padding_top = padding_h // 2
+                padding_bottom = padding_h - padding_top
+                pad = (padding_left, padding_right, padding_top,
+                       padding_bottom)
+                x = F.pad(x, pad, mode='constant', value=0.0)
+            kernel_size = (x.shape[-2] // size[-2], x.shape[-1] // size[-1])
```

Comment:
Suggest to keep naming rule consistent. Here we define `padding_right` with full name "right" but `padding_h` with Abbreviations `h`。It could be better to rename `padding_h` to `padding_heigth`  and `padding_w` to `padding_width`. Besides,  does `h_t` mean target height? If so, `target_height` could be better.